### ISSUE TITLE
Package 'cron-ext' requires a different Python: 3.12.3 not in '<3.12,>=3.7'

### ISSUE THREAD

**Issue by mistermax80** [meltano/cron-ext#141]
INFO: pip is looking at multiple versions of cron-ext to determine which version is compatible with other requirements. This could take a while.
ERROR: Package 'cron-ext' requires a different Python: 3.12.3 not in '<3.12,>=3.7'

**Comment by edgarrmondragon** [comment]
Yeah, we should probably bump the upper constraint or just remove, and maybe also drop some EOL versions to support only 3.9+.

I'll be AFK probably for the rest of the day, but I can probably review a PR on my phone, so PRs are welcome!

**Comment by mistermax80** [comment]
In my fork I removed the upper limit, but it is no enough, other limits are present:
2024-12-26T22:36:08,642 pip._internal.exceptions.UnsupportedPythonVersion: Package 'meltano-edk' requires a different Python: 3.12.6 not in '<3.12,>=3.7'
2024-12-26T22:36:08,645 Removed build tracker: '/private/var/folders/qw/wtx2z7yd3f76s8v7d61zyy2w0000gp/T/pip-build-tracker-8q6tk03e'

**Comment by edgarrmondragon** [comment]
@mistermax80 please give https://github.com/meltano/cron-ext/pull/142 a try

**Comment by mistermax80** [comment]
> [@mistermax80](https://github.com/mistermax80) please give [#142](https://github.com/meltano/cron-ext/pull/142) a try

It works!

Great!
